.github/workflows/golangci-lint.yml

```diff
@@ -14,8 +14,15 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
-        with:
-          version: v1.50.1
-          args: --verbose --timeout 10m --fix=false --new-from-rev=HEAD~ --config=.golangci.yml
```

Comment:
I kept this command separate from the one we run locally because using `--fix=true` in GitHub doesn't help because we can't pull those changes, and I'm also not sure it's compatible with `golangci/golangci-lint-action@v2` running. Locally, using `--fix=true` makes sense.